org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java

```diff
@@ -149,9 +160,23 @@ private String getMessage(IStatus status) {
 
 	}
 
-	private class ServerStatusMonitor extends NullProgressMonitor{
+	private List<String> getBundleList(Object initializationOptions) {
+		if (!(initializationOptions instanceof Map<?, ?>)) {
```

Comment:
We need to document the initialization parameters. I believe there is a "running jdt.ls" section in the main Readme.md where this could live.

Reply:
We should document this together with the extension point. Fred mention on wiki for extension point. For now, i'll update the "running jdt.ls" first.